src/connectedvmware/azext_connectedvmware/tests/latest/test_arcvmware_scenario.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure_devtools.scenario_tests import AllowLargeResponse
```

Comment:
```suggestion
from azure.cli.testsdk.scenario_tests import AllowLargeResponse
```

Reply:
Please use `azure.cli.testsdk` instead of `azure_devtools`. For more context, please refer to https://github.com/Azure/azure-cli-extensions/pull/4211